### ISSUE TITLE
Amélioration des arbres sur le fond de base

### ISSUE THREAD

**Issue by laem** [laem/cartes#615]
> https://github.com/laem/cartes/issues/521#issuecomment-2270470668
> Ainsi que les rangées d'arbres si possible (natural=tree_row): https://www.openstreetmap.org/way/1264562128

**Comment by u4y0u** [comment]
Les intérêts que je vois au 'rendu' des arbres:

1) La hauteur des arbres:
Un grand arbre est un point de repère visuel tout comme l'est un grand immeuble ou un château d'eau.
Il est donc essentiel en 'campagne' là ou le grand immeuble est absent.

2) le type de feuillage
Le principal intérêt ici est le repère couleur. Le needleleaved devient aussi un repère de carte en hiver.

les tags à prendre en compte serait donc height (ou est_height) et leaf_type.

Le leaf_cycle peut avoir un intérêt  si on peut changer le rendu et la couleur au fil des saisons. ça reste un plus qui pourrait avoir de la gueule :-)

**Comment by sylvain-m** [comment]
Désolé, je n'ai pas tout lu/suivi à ce sujet, mais comme je vois que tu réfléchis à l'affichage des arbres dans Cartes, je m'ajoute illico sur ce sujet pour être tenu au courant des évolution 😃 (et je vais de ce pas lire les liens que tu as partagé)
En effet, comme annoncé sur le Forum OSM France ([1][2]), je suis impliqué dans la cartographie des arbres dans OSM, et en particulier des arbres remarquables (labellisés ou non).
Pour ces arbres (et pas uniquement), je mets un maximum d'infos attributaires dans OSM, notamment l'espèce (avec `species:wikidata` en particulier), mais aussi la hauteur, la circonférence, l'année de plantation (ou âge estimé), etc...

Il est évident que toutes ces infos ne peuvent pas interagir sur le rendu, mais en tout cas, ce serait vraiment chouette si Cartes permettait, au clic sur les arbres, de les afficher.

C'est déjà prévu ?

[1] https://forum.openstreetmap.fr/t/arbres-remarquables/6342
[2] https://forum.openstreetmap.fr/t/arbres-labellises-arbre-remarquable-de-france/27333

**Comment by laem** [comment]
> ce serait vraiment chouette si Cartes permettait, au clic sur les arbres, de les afficher.

Oui, qu'appelles-tu "les afficher" ? Montrer une photo ? Quelle photo ? As-tu des exemples d'arbres, en URL, sur cartes.app ou osm.org ?

**Comment by sylvain-m** [comment]
Je parlais d'afficher les infos attributaires, c'est à dire les tags clé/valeurs, mais en format lisible par le grand public.
Concernant les photos, certains arbres ont des [wikimedia_commons](https://wiki.openstreetmap.org/wiki/Key:wikimedia_commons), mais c'est assez rares.
Exemple : https://www.openstreetmap.org/node/4783402443

**Comment by laem** [comment]
Tu peux afficher n'importe quelle entité OSM via cette URL https://cartes.app/?allez=arbre|n4783402443. 

n=node, etc.

![image](https://github.com/user-attachments/assets/78624c17-6f39-4f5e-ba96-9ed7129e1967)

**Comment by evinouze** [comment]
> Désolé, je n'ai pas tout lu/suivi à ce sujet, mais comme je vois que tu réfléchis à l'affichage des arbres dans Cartes, je m'ajoute illico sur ce sujet pour être tenu au courant des évolution 😃 (et je vais de ce pas lire les liens que tu as partagé) En effet, comme annoncé sur le Forum OSM France ([1][2]), je suis impliqué dans la cartographie des arbres dans OSM, et en particulier des arbres remarquables (labellisés ou non). Pour ces arbres (et pas uniquement), je mets un maximum d'infos attributaires dans OSM, notamment l'espèce (avec `species:wikidata` en particulier), mais aussi la hauteur, la circonférence, l'année de plantation (ou âge estimé), etc...
> 
> Il est évident que toutes ces infos ne peuvent pas interagir sur le rendu, mais en tout cas, ce serait vraiment chouette si Cartes permettait, au clic sur les arbres, de les afficher.
> 
> C'est déjà prévu ?
> 
> [1] https://forum.openstreetmap.fr/t/arbres-remarquables/6342 [2] https://forum.openstreetmap.fr/t/arbres-labellises-arbre-remarquable-de-france/27333

Je me greffe également à la discussion, suite à la lecture des nouveautés de cartes.app sur le forum OSM  et une navigation sur l'outil où je me disais : "ça serait trop cool de pouvoir afficher les arbres en 3D et présenter ça aux collègues des services des espaces verts...".
Car en effet, je travaille également à la contribution de la base OSM pour synchroniser le futur référentiel de suivi des arbres de la ville d'Échirolles via Wikidata, comme @sylvain-m l'a mentionné (et avec qui j'ai échangé sur [ce sujet](https://forum.openstreetmap.fr/t/choix-du-referentiel-taxonomique-pour-le-recensement-des-arbres/30328/8).

Si déjà il y avait la possibilité d'avoir la forme 3D, ça serait top. Et si en plus on peut gérer la forme et la couleur en fonction des attributs évoqués par @u4y0u , ca serait un argument de plus pour mettre la tête de Gooooogle sous l'eau :-)
